Server/sprawl_calculator/wup_calculator/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+import uuid
+
+# Create your models here.
+
+class Model(models.Model):
```

Comment:
This file needs to be removed from this PR, see the two pending comments at the bottom of this page so we can merge it.